File: test/java/test_PotentialTranspiration.java
//Test generation'

import java.math.BigDecimal;
public static class Test
{
    //first);
    //test1);

    public static void test1()
    {
        double evapoTranspiration = 830.958;
        double tau = 0.9983;

        PotentialTranspiration res0 = Estimation_PotentialTranspiration.CalculatePotentialTranspiration(evapoTranspiration,tau);
        System.out.println(" potentialTranspiration: "+res0.potentialTranspiration);

        System.out.println(((new BigDecimal(res0.potentialTranspiration)).setScale(3, BigDecimal.ROUND_HALF_DOWN)).equals((new BigDecimal(1.413)).setScale(3, BigDecimal.ROUND_HALF_DOWN)));
    }
}

File: test/cs/test_PotentialTranspiration.cs
'Test generation'

public static class Test
{
    //first);
    //test1

    public static void test1()
    {
        PotentialTranspiration res0 = Estimation_PotentialTranspiration.CalculatePotentialTranspiration(tau:0.9983,evapoTranspiration:830.958);

        Console.WriteLine("{0}","potentialTranspiration: "+res0.potentialTranspiration);

        Console.WriteLine("potentialTranspiration Comparison: ("+Math.Round(1.413,3)+";"+Math.Round(res0.potentialTranspiration,3)+") "+Equals(Math.Round(1.413,3),Math.Round(res0.potentialTranspiration,3)));
    }
}
